### PR TITLE
Add appInfo method

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog
```
Comment (robbiecarlton) [inline CHANGELOG.md:1]: I realize we haven't discussed any of this. I'm a big fan of keeping a changelog and semantic versioning, so throwing this in here in the hope that there's no objection, but totally understand if you want to discuss it more. I can just take it out of this PR in that case.

File: tests/unit/test_api.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const { Connection }			= require("../../src/index.js");
 
 describe("Javascript API", () => {
 
-    it("should call zome function", async () => {
+    it("should call COMB.call", async () => {
```
Comment (robbiecarlton) [inline tests/unit/test_api.js:15]: Just noticed this test name was a little misleading. It doesn't test as much as that name implies.

**Review by mjbrisebois — CHANGES_REQUESTED** [review]
I am still waiting for responses to my comments from 5 days ago on the planning document https://hackmd.io/QfADWyIRRyy8TYRL6Gge8g

If we decide that this is something we want to implement, then we have several other issues to address before the code review.  For example, this work is under B-03636 which (based on the AC) is blocked by B-03313.  We could avoid that blocker by implementing it for the Chaperone development server first and implement it for Hosting later.  However, I am still of the opinion that we should not implement any admin calls in the Web SDK.  Please address the planning document comments before requesting my code review.

**Comment by robbiecarlton** [comment]
> I am still waiting for responses to my comments from 5 days ago
  I've been out of the office for the last 5 days
I've now replied to your comments.

But based on my work on chaperone, I'm not sure this is the right approach, so I'm putting this back to a draft PR for now. It's looking like `web-sdk` might not need any changes.

**Pull request closed without merging**